src/app/core/services/proof.service.ts

```diff
@@ -13,11 +13,17 @@ import { GeolocationService } from './geolocation.service';
   providedIn: 'root'
 })
 export class ProofService {
+  enableGeolocation: string = 'disable'; 
 
   constructor(
     private readonly geolocationService: GeolocationService,
   ) { }
 
+  async setLocationInfoCollection(enable: string) {
+    console.log("setLocationInfoCollection",enable);
+    return this.enableGeolocation=enable;
```

Comment:
```suggestion
    this.enableGeolocation = enable;
```
No need to return anything (the return value is not used)